components/engine/daemon/graphdriver/aufs/aufs.go

```diff
@@ -50,6 +50,8 @@ import (
 
 	rsystem "github.com/opencontainers/runc/libcontainer/system"
 	"github.com/opencontainers/selinux/go-selinux/label"
+	"github.com/pkg/errors"
+	"github.com/vbatts/tar-split/tar/storage"
```

Comment:
You don't need to add this here, it's already added on line 40.

It's causing this error in the build jobs:
```
00:02:39.788 [init] ---> Making bundle: binary (in bundles/17.06.1-ce-rc1/binary)
00:02:40.046 [init] Building: bundles/17.06.1-ce-rc1/binary-daemon/dockerd-17.06.1-ce-rc1
00:03:36.261 [init] # github.com/docker/docker/daemon/graphdriver/aufs
00:03:36.261 [init] daemon/graphdriver/aufs/aufs.go:54: storage redeclared as imported package name
00:03:36.261 [init] 	previous declaration at daemon/graphdriver/aufs/aufs.go:40
```